CITATION.cff

```diff
@@ -330,6 +330,11 @@ authors:
     family-names: Smeets
     given-names: Stef
     orcid: "https://orcid.org/0000-0002-5413-9038"
+  -
+    affiliation: "Netherlands eScience Center"
+    family-names: Siqueira
+    given-names: Abel Soares
```

Comment:
```suggestion
    family-names: Soares Siqueira
    given-names: Abel
```
Thanks for adding me! Can you update my `family-names`, please?

Reply:
ah Spaniards and their 14 family names :rofl: